media-gfx/noteshrink/noteshrink-0.1.1.recipe

```diff
@@ -3,28 +3,54 @@ DESCRIPTION="Noteshrink is an utility that is used to convert scans of notes to
 HOMEPAGE="https://github.com/mzucker/noteshrink"
 COPYRIGHT="2016 Matt Zucker"
 LICENSE="MIT"
-REVISION="1"
+REVISION="2"
 SOURCE_URI="https://github.com/mzucker/noteshrink/archive/$portVersion.tar.gz"
 CHECKSUM_SHA256="2a29c09768973e688b693b61337c6c49384e8123cf88824dba335cc8c4ed2ca8"
 
-ARCHITECTURES="!any"
-# Doesn't work on gcc2hybrid - missing dependencies.
+ARCHITECTURES="any"
+# The package is "any" because the result is only a Python script under /bin/.
+# However, on gcc2 systems, it will depend on Python modules only available for _x86.
+if [ "$targetArchitecture" = x86_gcc2 ]; then
+	hostArchSuffix="_x86"
+else
+	hostArchSuffix=""
+fi
```

Comment:
Why insist on keeping this any? I don't think the 32bit package will run on 64bit (or does it)?

Reply:
So basically it can't be `any` then?

Reply:
As long as pillow and scipy aren't fixed, it can't be `any`. But they should be fixed, similar to numpy.

Reply:
Library dependencies will be hard to fix for pillow and numpy (for gcc2), so I guess we're stuck on secondary architecture for 32bit.

Reply:
No, they should just add a block like this to the `eval` block:

```sh
if [ $targetArchitecture = x86_gcc2 ]; then\
   PROVIDES_${pythonPackage}+=\"\n\
       scipy_${pythonPackage} = $portVersion\n\
       \"\n\
fi\n\
```

Reply:
I don't think even that will make it install-able on 64bit? As nothing will provide `lib:liblapack$secondaryArchSuffix` (numpy) for instance?

Reply:
It will still depend on scipy_x86_python39 on 32 bit, but that package will provide both `scipy_x86_python39` and `scipy_python39`, so that the `x86` can be dropped from the requirements of `any` packages.

Reply:
OK, starts to make sense, just wanted to grasp the changes, at least at it's current state it doesn't work. :)